networking.adoc

```diff
@@ -0,0 +1,1081 @@
+
+[source,terminal]
+----
+$ oc patch svc <name> -p \
+  '{
+    "spec": {
+      "externalIPs": [ "<ip_address>" ]
+    }
+  }'
+----
+
+
+[source,terminal]
+----
+$ oc patch svc mysql-55-rhel7 -p '{"spec":{"externalIPs":["192.174.120.10"]}}'
+----
+
+## DNS Operator in {product-title}
+
+[source,terminal]
+----
+$ oc edit dns.operator/default
+----
+
+## Configuring the cluster-wide proxy
+
+[source,terminal]
+----
+$ oc create -f user-ca-bundle.yaml
+----
+
+
+[source,terminal]
+----
+$ oc edit proxy/cluster
+----
+
+
+[source,terminal]
+----
+$ oc edit proxy/cluster
+----
+
+## Adding a pod to an SR-IOV additional network
+
+[source,terminal]
+----
+$ oc create -f <name>.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f <filename> 
+----
+
+## Configuring an SR-IOV network device
+
+[source,terminal]
+----
+$ oc create -f <name>-sriov-node-network.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create sriovnetwork.openshift.io cluster
+----
+
+## Configuring an SR-IOV InfiniBand network attachment
+
+[source,terminal]
+----
+$ oc create -f <name>.yaml
+----
+
+## Configuring an SR-IOV Ethernet network attachment
+
+[source,terminal]
+----
+$ oc create -f <name>.yaml
+----
+
+## Configuring the SR-IOV Network Operator
+
+[source,terminal]
+----
+$ oc patch sriovoperatorconfig default \
+  --type=merge -n openshift-sriov-network-operator \
+  --patch '{ "spec": { "enableInjector": <value> } }'
+----
+
+
+[source,terminal]
+----
+$ oc patch sriovoperatorconfig default --type=merge \
+  -n openshift-sriov-network-operator \
+  --patch '{ "spec": { "enableOperatorWebhook": <value> } }'
+----
+
+
+[source,terminal]
+----
+$ oc patch sriovoperatorconfig default --type=json \
+  -n openshift-sriov-network-operator \
+  --patch '[{
+      "op": "replace",
+      "path": "/spec/configDaemonNodeSelector",
+      "value": {<node-label>}
+    }]'
+----
+
+## Using virtual functions (VFs) with DPDK and RDMA modes
+
+[source,terminal]
+----
+$ oc create -f intel-dpdk-node-policy.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f intel-dpdk-network.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f intel-dpdk-pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-dpdk-node-policy.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-dpdk-network.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-dpdk-pod.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-rdma-node-policy.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-rdma-network.yaml
+----
+
+
+[source,terminal]
+----
+$ oc create -f mlx-rdma-pod.yaml
+----
+
+## Ingress Operator in {product-title}
+
+[source,terminal]
+----
+$ oc --namespace openshift-ingress create secret tls custom-certs-default --cert=tls.crt --key=tls.key
+----
+
+
+[source,terminal]
+----
+$ oc patch --type=merge --namespace openshift-ingress-operator ingresscontrollers/default \
+  --patch '{"spec":{"defaultCertificate":{"name":"custom-certs-default"}}}'
+----
```

Comment:
Requires a declarative alternative

Reply:
PR addressing this: https://github.com/openshift/openshift-docs/pull/32924